.github/workflows/test.yml

```diff
@@ -145,7 +146,7 @@ jobs:
           npm run load
 
       - name: Execute
-        run: docker-compose exec -T ${{ matrix.services }} npm run api-test
+        run: docker-compose exec -T ${{ matrix.services }} npm run api-test --workspace=${{ matrix.services }}
```

Comment:
how does this work for services (like libs) that will only have unit tests and not api-tests?